todo_app/data/logging.py

```diff
@@ -0,0 +1,32 @@
+import logging
+import os
+from loggly.handlers import HTTPSHandler
+from logging import Formatter, StreamHandler
+
+LOGGER_NAME = "todo_app"
+
+def setupLogger():
+    streamHandler = StreamHandler()
+
+    handlers = [streamHandler]
+    if os.getenv("LOGGLY_TOKEN") is not None:
+        logglyHandler = HTTPSHandler(f'https://logs-01.loggly.com/inputs/{os.getenv("LOGGLY_TOKEN")}/tag/todo-app')
+        logglyHandler.setFormatter(Formatter("[%(asctime)s] %(levelname)s in %(module)s: %(message)s"))
+        handlers.append(logglyHandler)
+
+    logLevel = os.environ.get('LOG_LEVEL')
+    if logLevel == None :
+        logLevel = "INFO"
```

Comment:
You can pass a default value through as a second argument to `.get` (which is the same on a dictionary, if you ever want to access a value safely or with a default) so we could reduce this to:
```suggestion
    logLevel = os.environ.get('LOG_LEVEL', 'INFO')
```